nmtwizard/framework.py

```diff
@@ -1597,3 +1612,12 @@ def is_joint_vocab(vocabulary_config):
     source = vocabulary_config.get("source", {})
     target = vocabulary_config.get("target", {})
     return source.get("path") == target.get("path")
+
+
+def add_target_to_score_output(target_path, score_path):
+    with open(target_path) as tf, open(score_path) as sf:
+        scores = [x.split(" ||| ")[0] for x in sf.readlines()]
+        target_lines = tf.readlines()
```

Comment:
The file to score can be quite big. Can you look if you can read and copy the target line by line? It is OK to write the result in another file.